udata_hydra/migrations/main/20230121_rev7_up_rev8.sql

```diff
@@ -0,0 +1,4 @@
+-- drop csv_detective column (will be stored alongside csv tables)
```

Comment:
I stopped using it as soon as Migrator was ready https://github.com/etalab/udata-hydra/pull/50/files#diff-f41a6470394a623e21d04beea2296b4f68d1d6141cf999061069774de1b09716 https://github.com/etalab/udata-hydra/pull/50/files#diff-8b46a3ae1589e25a7a22eda698aa2c35589422fba7f9ac74f0ab38f353f1f962 — but yeah this one could be renamed. I'm just a bit lazy because I would have to rename stuff locally and somehow having "drop_migrations" as the first non-rev file kinda makes sense. No? 😇

Reply:
Let's say so ! :p